Readme.md

```diff
@@ -64,6 +64,13 @@ address: "127.0.0.1"    # the bind address
 port: "8000"            # the listening port
 dir: "/home/webdav"     # the provided base dir
 prefix: "/webdav"       # the url-prefix of the original url
+deny:                   # deny your OS to create garbage
+  create:
+    file:               # deny creation of specified files
+      - .DS_Store
+      - ._*             # globbing supported, https://pkg.go.dev/path/filepath#Match
+  directory:            # deny creation of specified directories
```

Comment:
I think there are missing leading whitespaces? 🤔

Reply:
sorry for late reply, I seem to have missed the notice.

Nice find, will fix ASAP.